trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -147,7 +150,22 @@ base16(<SpanId>) = 00f067aa0ba902b7
 base16(<TraceOptions>) = 00  // not-sampled
 ```
 
-# TraceState field
+## Versioning
+
+Implementation is opinionated about future version of specification. We hope to make format of `traceparent` additive.
+
+Implementation should follow a following rules when parsing headers with unexpected format:
+
+1. Pass thru services should not analyze version. Pass thru service needs to expect that headers may have bigger limits in future.
+2. When version prefix cannot be parsed (it's not 2 hex characters followed by dash), implementation should restart trace.
+3. If higher version is detected - implementation should try to parse.
+  a. If size of header shorter - implementation should not parse it.
```

Comment:
shorter than what, a combined length of 2+32+16+2+(separators)?

Reply:
yes. Made it explicit